NETDefault/WebApp/web.config

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <system.webServer>
+    <handlers>
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
+    </handlers>
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" hostingModel="inprocess" />
+    <httpProtocol>
+      <customHeaders>
+        <remove name="X-Powered-By" />
+      </customHeaders>
+    </httpProtocol>
+    <applicationInitialization>
+      <add initializationPage="/api/cachedependent/check" />
```

Comment:
This gets difficult to read; perhaps _/api/cache-dependent/check_ as endpoint?

Also it would be clean if we could find a resource based way to describe this URL:
**GET /api/caches** -> List with cache keys
**GET /api/caches/:id** -> Return cache object,  with resource based URL for possibility to add multiple different sections of caches or actual cache keys, depending on scenario
**GET /api/caches/:id/check** -> special action, in this case a simple quick return when cache is present might be ok
**DELETE /api/caches/:id**

Or am I possibly misunderstanding the terminology here?

Reply:
I think you are misunderstanding the terminology. The CacheDependentController is just a sample controller which relies on cache, and not a cache controller with all kinds of features. I'll rename it to make this more clear.